packages/venia-ui/lib/components/Image/image.css

```diff
@@ -16,18 +16,13 @@
      */
 }
 
-.loaded {
+.image_positioned {
     display: block;
     position: absolute;
     top: 0;
     left: 0;
 }
 
-.notLoaded {
-    composes: loaded;
-    display: none;
```

Comment:
I feel like the code deserves a TODO or something. Instead of changing the classes applied, perhaps we could just comment out this line and add a comment here and in the `onLoad` callback about it not functioning as expected.

Reply:
Or perhaps this is the right approach and we should always just overlay the loaded image over the placeholder...Hmm

Reply:
Just started seeing this happen in Chrome this week with our logo (which uses `Image`). Note: we are using ~v4 code currently.

FYI: overlaying probably is not safe. We have a transparent background on some assets (i.e. brand logos) passed into `Image`.

Reply:
The overlay is always onto a transparent background placeholder (unless a different placeholder is explicitly passed in), so no worries there.

Reply:
Outdated.